addons/web/static/tests/env_tests.js

```diff
@@ -173,3 +179,66 @@ QUnit.test("get an object containing dependencies as second arg", async (assert)
     });
     await makeTestEnv();
 });
+
+QUnit.test(
+    "startServices: throws if all dependencies are not met in the same microtick as the call",
+    async function (assert) {
+        assert.expect(3);
+        clearRegistryWithCleanup(serviceRegistry);
+        clearServicesMetadataWithCleanup();
```

Comment:
@aab-odoo I suspect this will be problematic during the rebase, since the commit this fixup targets is older than the one that introduces SERVICES_METADATA clearing. I thought I had this line in the fixup commit, did you move them or did I not do what I think I did?

Reply:
Je sais, je me suis dit que je verrai bien au moment de clean la branche ^^